chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -223,21 +244,32 @@ def forward_gpu(self, inputs):
             ret = ret.reshape(t.shape)
         return ret,
 
+    def _soft_target_loss(self, xp, x, t, log_y):
+        kl_d = xp.sum(t * (xp.log(t + self.eps) - log_y), axis=1)
```

Comment:
To compute the cross entropy, 
```suggestion
        ____ = -xp.sum(t * log_y, axis=1)
```

Reply:
Now, I'm wondering if this is correct, since it fails at the test below. 
https://github.com/chainer/chainer/blob/1ab26322767bec5fe9b0c6b5da702b95cb1ef3bb/tests/chainer_tests/functions_tests/loss_tests/test_softmax_cross_entropy.py#L618
This test uses output of softmax as soft target label, so output of softmax_cross_entropy becomes zero or almost zero. But it becomes non-zero when 'cross-entropy' is used as soft target loss calculation.

Reply:
Done in 1ab26322767bec5fe9b0c6b5da702b95cb1ef3bb.